backend/main.py

```
@@ -1005,6 +978,122 @@ async def get_user_profile(user_id: int, db: Session = Depends(get_db)):
+            "session_timeout_minutes": ACCESS_TOKEN_EXPIRE_MINUTES
+        }
+    except Exception as e:
+        logger.error(f"Error getting session status: {e}")
+        return {
+            "authenticated": False,
+            "error": "Failed to get session status"
+        }
+
+@app.post("/users/session/refresh")
+async def refresh_user_session(
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)